### PR TITLE
1.0: Add description about new dump function of fluent-ctl and RPC

### PR DESCRIPTION
**Pull request by daipom**
For https://github.com/fluent/fluentd/pull/3680

### CODE REVIEW

**Review by cosmo0920 — APPROVED** [review]

**Comment by daipom** [comment]
Removed the description about RPC of this feature.

**Comment by daipom** [comment]
Removed the description about RPC of this feature.

File: deployment/command-line-option.md [hunk 1]
```diff
@@ -161,21 +161,55 @@ Options:
...
+
 ```text
 fluent-ctl shutdown 11111
 ```
 
+If you run Fluentd as a Windows service, then you have to specify the svcname (the name of the Windows service).
+
+```text
+fluent-ctl dump fluentdwinsvc
+```
```
Comment (ashie) [inline deployment/command-line-option.md]: The service name isn't mandatory required.
On windows, `fluentdwinsvc` is specified by default when the 3rd argument is omitted.

**Review by ashie — CHANGES_REQUESTED** [review]

File: deployment/command-line-option.md [hunk 2]
```diff
@@ -161,21 +161,55 @@ Options:
 Control `fluentd` process using [Signals](signals.md) or Windows Event.
 
 ```text
-Usage: fluent-ctl COMMAND PID
+Usage: fluent-ctl COMMAND PID_OR_SVCNAME
```
Comment (ashie) [inline deployment/command-line-option.md]:
```suggestion
Usage: fluent-ctl COMMAND [PID_OR_SVCNAME]
```

Adding `[ ... ]` is recommended for `PID_OR_SVCNAME` to indicate it's an omittable argument.

Here is the actual help string of `fleunt-ctl` on Windows:
```console
aho@SKYLAKE C:\Users\aho\Projects\fluentd>bundle exec fluent-ctl

Usage: C:/Users/aho/Projects/fluentd/vendor/bundle/ruby/2.7.0/bin/fluent-ctl COMMAND [PID_OR_SVCNAME]

Commands:
  shutdown
  restart
  flush
  reload
  dump

Error: Command isn't specified!
```

**Comment by daipom** [comment]
Thank you for the comment, I will fix it soon,

File: deployment/command-line-option.md (hunk 1 above)
Comment (daipom) [reply deployment/command-line-option.md]: Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/397/commits/dc0ddcfc33deb68577738093d0d519d23bf1c68f

File: deployment/command-line-option.md (hunk 2 above)
Comment (daipom) [reply deployment/command-line-option.md]: Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/397/commits/dc0ddcfc33deb68577738093d0d519d23bf1c68f

**Comment by daipom** [comment]
I fixed them in https://github.com/fluent/fluentd-docs-gitbook/pull/397/commits/dc0ddcfc33deb68577738093d0d519d23bf1c68f

**Review by ashie — APPROVED** [review]

**Comment by ashie** [comment]
Thanks!

**Pull request merged**